sentry_sdk/integrations/django/caching.py

```diff
@@ -0,0 +1,101 @@
+import functools
+from typing import TYPE_CHECKING
+
+from django import VERSION as DJANGO_VERSION
+from django.core.cache import CacheHandler
+
+from sentry_sdk import Hub
+from sentry_sdk.consts import OP, SPANDATA
+from sentry_sdk._compat import text_type
+
+
+if TYPE_CHECKING:
+    from typing import Any
+    from typing import Callable
+
+
+METHODS_TO_INSTRUMENT = [
+    "get",
+    "get_many",
+]
+
+
+def _instrument_call(cache, method_name, original_method, args, kwargs):
+    # type: (CacheHandler, str, Callable[..., Any], Any, Any) -> Any
+    description = "{} {}".format(method_name, " ".join(args))
+
+    with Hub.current.start_span(op=OP.CACHE, description=description) as span:
```

Comment:
you need a hub/integration check before here too because this is during execution of each function and not just during `setup_once` in the beginning.

```python
        hub = Hub.current
        integration = hub.get_integration(DjangoIntegration)
        if integration is None or not integration.cache_spans:
            return None
```

see template example
https://github.com/getsentry/sentry-python/blob/6c7625c1ae7cae1a81761f7576a59f00556d15c2/sentry_sdk/integrations/django/templates.py#L87-L96

Reply:
got it